handlers/LiveAgentSessionHandler.ts

```diff
@@ -48,4 +56,37 @@ export class LiveAgentSession {
 			console.error(ErrorLogs.LIVEAGENT_SESSION_CLASS_FAILED, error);
 		}
 	}
+
+	private endChat =  async (
```

Comment:
Instead of creating a new function, we can reuse `handleEndChat` from `HandleEndChatCallback.ts`